types/yup/index.d.ts

```diff
@@ -351,12 +351,27 @@ export interface TestOptions<P extends Record<string, any> = {}, R = any> {
     exclusive?: boolean;
 }
 
+export interface SchemaFieldRefDescription {
+  type: 'ref';
```

Comment:
minor nit: can you keep the indentation consistent with the rest of the file?

Reply:
oops.